### ISSUE TITLE
[BUG] Refine CLI not supported in yarn monorepos

### ISSUE THREAD

**Issue by itsjavi** [refinedev/refine#3732]
### Describe the bug

Since yarn workspaces use the root `node_modules` dir of the project instead of individual ones per package, 
refine CLI fails to resolve the `.bin` directory, because it's expected to be in the current directory relative to the package.json where `refine` is called.

The problem seems to be here https://github.com/refinedev/refine/blob/next/packages/cli/src/commands/runner/projectScripts.ts#L3

It fails to find `./node_modules/.bin/next`
```
@app/admin:dev: Error: Command failed with ENOENT: /usr/src/app/packages/admin/node_modules/.bin/next dev
@app/admin:dev: spawn /usr/src/app/packages/admin/node_modules/.bin/next ENOENT
```

### Steps To Reproduce

1. Create a new refine nextjs app, called e.g. `my-admin`
2. Remove the package-lock.json and node_modules of the new refine app
3. Move it inside a yarn monorepo (Yarn Berry or not, I think it's not important)
4. Run `yarn install`
5. Run `yarn workspace my-admin run refine dev`

### Expected behavior

The NextJS app should be started in dev mode.

**Comment by itsjavi** [comment]
Thank you for checking, it's also not possible to use other CLI commands like `swizzle`, unless I temporarily install dependencies in the workspace package itself.

Adding a general option to the CLI to specify the path of the `node_modules` dir could be a solution IMHO.

**Comment by omeraplak** [comment]
Hey @itsjavi ,
We've released [@refinedev/cli@2.5.0](https://github.com/refinedev/refine/releases/tag/%40refinedev%2Fcli%402.5.0) with monorepo support ⚡️ Could you approve this fix?
